mod/wikirate/spec/set/right/wikirate_article_spec.rb

```diff
@@ -13,8 +13,8 @@
         article = Card.new :name=>"#{@company.name}+#{@topic.name}+#{Card[:wikirate_article].name}",:type=>"basic"
         html = article.format.render_missing
 
-        expect(html).to have_tag("div",:with=>{:class=>"claim-tip"}) do
-          with_tag "textarea",:with=>{:id=>"citable_claim"},:text=>/#{@citation}/
+        expect(html).to have_tag("div",:with=>{:class=>"note-tip"}) do
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space missing after comma.<br>Surrounding space missing for operator `=>`.<br>Space inside { missing.<br>Space inside } missing.